scripts/install2

```diff
@@ -77,6 +81,10 @@ if [ "${#MESH_NAME}" == 0 ]; then
     export MESH_NAME="tomesh"
 fi
 
+if [[ $RPI_REVISION == "OPI" ]]; then
```

Comment:
All the OPI-specific stuff should go into one single blob. This will quickly become unmanageable with `$RPI_REVISION == X` scattered all over the files.